app/views/shared/_navbar.html.erb

```diff
@@ -11,54 +11,53 @@
       <% end %>
 
       <% if user_signed_in? %>
-        <div class="nav-item dropdown navbar_dropdown_item h-100 px-2 rounded-0 border-0">
-          <a class="nav-link h-100 d-flex align-items-center fs-5" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-            Base
-          </a>
-          <ul class="dropdown-menu bg-primary rounded-0 navbar_dropdown_item">
-            <li>
-              <%= link_to "Home", home_path, class: 'dropdown-item navbar_dropdown_item' if policy(:static_page).home? %>
-            </li>
-            <li>
-              <%= link_to "Zones", zones_path, class: 'dropdown-item navbar_dropdown_item' if policy(Zone).index? %>
-            </li>
-            <li>
-              <%= link_to "Rooms", rooms_path, class: 'dropdown-item navbar_dropdown_item' if policy(Room).index?%>
-            </li>
-          </ul>
-        </div>
-      <% end %>
-
-      <% if user_signed_in? %>
-        <div class="nav-item dropdown navbar_dropdown_item h-100 px-2 rounded-0 border-0">
-          <a class="nav-link h-100 d-flex align-items-center fs-5" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-            Admin
-          </a>
-          <ul class="dropdown-menu bg-primary rounded-0 navbar_dropdown_item">
-            <li>
-              <%= link_to "Common Attributes", common_attributes_path, class: 'dropdown-item navbar_dropdown_item' if policy(CommonAttribute).index? %>
-            </li>
-            <li>
-              <%= link_to "Create Preferences", app_preferences_path, class: 'dropdown-item navbar_dropdown_item' if policy(AppPreference).index? %>
-            </li>
-            <li>
-              <%= link_to "App Preferences", configure_prefs_path, class: 'dropdown-item navbar_dropdown_item' if policy(AppPreference).configure_prefs? %>
-            </li>
-            <li>
-              <%= link_to "Announcements", announcements_path, class: 'dropdown-item navbar_dropdown_item' if policy(Announcement).update? %>
-            </li>
-            <li>
-              <%= link_to "Rovers", rovers_path, class: 'dropdown-item navbar_dropdown_item'if policy(Rover).index? %>
-            </li>
-          </ul>
-        </div>
+        <% unless is_rover?(current_user) %>
```

Comment:
Same reasoning as the previous comment. If an admin is also a rover, they don't see any of the links in the navbar. Doing `<% if is_admin?(current_user) %>` might be better.